packages/blockbook/src/controller.ts

```diff
@@ -277,9 +277,9 @@ export class Blockbook extends Controller {
     },
   })
   @Get('tx-specific/{txid}')
-  async getTransactionSpecific(@Path() txid: string): Promise<TxSpecific> {
+  async getTransactionSpecific(@Path() txid: string): Promise<Tx> {
     try {
-      const { data } = await this.instance.get<TxSpecific>(`api/v2/tx-specific/${txid}`)
+      const { data } = await this.instance.get<Tx>(`api/v2/tx-specific/${txid}`)
```

Comment:
These should remain `TxSpecific` and we will need to add each chain's specific interface. In this case we can add `BitcoinTxSpecific`

Reply:
yep all UtxoResponse is removed now